### ISSUE TITLE
Autosleep problem

### ISSUE THREAD

**Issue by oldhasbeen** [koreader/koreader#2860]
* KOReader version: 1023
* Device: Aura H2O

#### Steps to reproduce
turn on - wait until autosleep should start - 
my last install - 994 - did autosleep ok. I copied my previous reader settings  from 994 to 1023 but no autosleep there. I always change in frontend/uimanager autosleep time from 60 minutes to 5. I am back to 994 now and autosleep is working there.
I wonder if the battery drain issue as reported in #2854 is linked to this.

**Comment by KenMaltby** [comment]
My autosleep settings migrated from unimanager.lua to also appear in settings.reader.lua .  It is still working in 1023, for me. (Although I updated/installed it with the unimanager change, as well.)

**Comment by oldhasbeen** [comment]
@KenMaltby 
I have repeated the 1023 installation procedure and the autosleep problem remained. Since yours works I deleted the koreader completely from my device, extracted 1023 anew time from the zip container and reinstalled in the usual way. Autosleep works now ok. I have no smart explanation for this other than a "glitch" or somewhere garbage data or, or ....
I leave this issue open for some more days but will close it if the problem does not occur again or others report it too.
Thanks for your info - I would not have persisted and would have thought that there is a new problem. At this stage it looks however that it is just another one of "those things".
